certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json

```diff
@@ -20,321 +20,6 @@
   ],
   "produces": [],
   "paths": {
-    "/ApiEnterprise/SearchEnterprise": {
-      "post": {
-        "responses": {
-          "default": {
-            "description": "default"
-          },
-          "200": {
-            "description": "success",
-            "schema": {
-              "type": "object",
-              "properties": {
-                "Message": {
-                  "type": "string",
-                  "description": "Message"
-                },
-                "Success": {
-                  "type": "boolean",
-                  "description": "Success"
-                },
-                "EnterpriseId": {
-                  "type": "array",
-                  "items": {
-                    "type": "integer",
-                    "format": "int32"
-                  },
-                  "title": "Enterprise ID of the requested business partner."
-                }
-              }
-            }
-          },
-          "303": {
-            "description": "Endpoint is obsolete. Please use the endpoint GetEnterprises",
-            "schema": {
-              "type": "object",
-              "properties": {
-                "Message": {
-                  "type": "string",
-                  "description": "Message"
-                },
-                "Success": {
-                  "type": "boolean",
-                  "description": "Success"
-                },
-                "EnterpriseId": {
-                  "type": "array",
-                  "items": {
-                    "type": "integer",
-                    "format": "int32"
-                  },
-                  "title": "Enterprise ID of the requested business partner."
-                }
-              }
-            }
-          }
-        },
-        "summary": "[Obsolete] Search Enterprise ID",
-        "description": "Search the Tesseron enterprise ID with the customer number.",
-        "operationId": "SearchEnterprise",
-        "parameters": [
-          {
-            "name": "body",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "type": "object",
-              "properties": {
-                "referenceNumber": {
-                  "type": "string",
-                  "description": "Please provide the business partner ID",
-                  "title": "Business Partner ID"
-                }
-              },
-              "required": [
-                "referenceNumber"
-              ]
-            }
-          }
-        ]
-      }
-    },
-    "/ApiEnterprise/GetEnterprise": {
```

Comment:
Can you please confirm that the existing flows will not break the flows?

The operation has been deleted. This change breaks existing flows. ValuePath: operation/GetEnterprise